app/views/home/index.html.haml

```diff
@@ -157,7 +163,9 @@
       Looking to
       .orange
         help?
-      = btn_link_to 'Volunteer', volunteer_link
+      .volunteer-buttons
+        = btn_link_to 'Volunteer', volunteer_link
+        = btn_link_to 'Be a Mentor', mentor_link
```

Comment:
Getting mentors has traditionally been a harder thing to do, might want to consider adding this to the top of the page too